lib/landrush/command.rb

```diff
@@ -63,7 +67,7 @@ def help; <<-EOS.gsub(/^      /, '')
         set { <host> <ip> | <alias> <host> }
           adds the given host-to-ip or alias-to-hostname mapping.
           existing host ip addresses will be overwritten
-        rm, del { <host> | <alias> }
+        rm, del { <host> | <alias> | --all }
```

Comment:
> What we're looking for is actually either or, not both. Right?

You are right. It makes sense as you describe it. I guess it looks a bit unusual.